data.json

```diff
@@ -127,4 +127,12 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
   }
```

Comment:
You are missing a coma (`,`) in here.

Reply:
I'm sorry, but I'm not sure where one is supposed to be.

Reply:
```suggestion
  },
```